driver/command/command.go

```diff
@@ -0,0 +1,105 @@
+package command
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/deislabs/cnab-go/driver"
+)
+
+// Driver relies upon a system command to provide a driver implementation
+type Driver struct {
+	Name string
+}
+
+// Run executes the command
+func (d *Driver) Run(op *driver.Operation) error {
+	return d.exec(op)
+}
+
+// Handles executes the driver with `--handles` and parses the results
+func (d *Driver) Handles(dt string) bool {
+	out, err := exec.Command(d.cliName(), "--handles").CombinedOutput()
+	if err != nil {
+		fmt.Printf("%s --handles: %s", d.cliName(), err)
+		return false
+	}
+	types := strings.Split(string(out), ",")
+	for _, tt := range types {
+		if dt == strings.TrimSpace(tt) {
+			return true
+		}
+	}
+	return false
+}
+
+func (d *Driver) cliName() string {
+	return "cnab-" + strings.ToLower(d.Name)
+}
+
+func (d *Driver) exec(op *driver.Operation) error {
+	// We need to do two things here: We need to make it easier for the
+	// command to access data, and we need to make it easy for the command
+	// to pass that data on to the image it invokes. So we do some data
+	// duplication.
+
+	// Construct an environment for the subprocess by cloning our
+	// environment and adding in all the extra env vars.
+	pairs := os.Environ()
+	added := []string{}
+	for k, v := range op.Environment {
+		pairs = append(pairs, fmt.Sprintf("%s=%s", k, v))
+		added = append(added, k)
+	}
+	// DUFFLE_VARS is a list of variables we added to the env. This is to make
+	// it easier for shell script drivers to clone the env vars.
+	pairs = append(pairs, fmt.Sprintf("DUFFLE_VARS=%s", strings.Join(added, ",")))
```

Comment:
Should we change `DUFFLE_VARS` to a different name?  Or, is this still useful to keep around?

Reply:
Good catch, updated it.